api-tests/data/cri-stub-requests/dcmaw/kenneth-changed-given-name-passport-valid/credentialSubject.json

```diff
@@ -11,7 +11,7 @@
       "nameParts": [
         {
           "type": "GivenName",
-          "value": "Michael"
+          "value": "Ken"
```

Comment:
Is there a reason you changed this? I think it's probably clearer to use a totally different name(?)

I guess it's a bit odd that it's under `kenneth-changed-given-name` if the name is `Michael` 🤔

Reply:
This was just dedupliucation - we had two different "change name" cases - Ken and Michael. Happy to revert to Michael if it's more different, but I'm not sure it should matter too much.